### DISCUSSION TITLE
A permission setting

### DISCUSSION THREAD

**Discussion by Jhesterccj** [Zrips/CMI#7920] [Feature Requests]
Functions that prevent players from destroying, placing, etc
For example, ess's
essentials.build.destroy.*
essentials.build.drop.*
essentials.build.interact.*
essentials.build.pickup.*
essentials.build.place.*
essentials.build.use.*

**Comment by CloudeLecaw** [comment]
isnt that the job ob your protection plugin ?

if your protection plugin doesnt handle this then you might should change it...

well at least build, use and break protection is out of the scope of cmi.
